### PR TITLE
Update decision making to new IAM owners

### PR DESCRIPTION
**Pull request by gene1wood**

### CODE REVIEW

File: DECISION-MAKING.md [hunk 1]
```diff
@@ -161,22 +135,15 @@ Repository:
...
 - Peers: @gcox
 - Operator: CoreServices
 
 ## DuoSecurity (2FA)
 
 This is the 2FA/MFA product utilized to augment OpenLDAP with a second-factor.
-
-- Owner: @gene1wood
-- Peers: @april @tristanweir
-- Operator: Third party (Duo)
```
Comment (floatingatoll) [inline DECISION-MAKING.md:182]: I think we should keep this Operator line.

**Review by floatingatoll — CHANGES_REQUESTED** [review]

File: DECISION-MAKING.md (hunk 1 above)
Comment (gene1wood) [reply DECISION-MAKING.md:182]: Sure, thing, added it back in.

**Review by floatingatoll — COMMENTED** [review]
No other comments at this time.

**Comment by sciurus** [comment]
I have an alternative to this in #118

**Pull request closed without merging**